src/main/java/nopol_examples/nopol_example_1/NopolExample.java

```diff
@@ -9,11 +9,11 @@
 	 */
 	public char charAt(String s, int index) {
 		
-		if ( index ==  0 ) // Fix index <= 0
-			return s.charAt(0);
+		if ( index == 0 ) // Fix index <= 0
+			return s.charAt(index);
 		
 		if ( index < s.length() )
-			return s.charAt(index);
+			return s.charAt(s.length()-1);
```

Comment:
This line (16) has been identified with a suspiciousness value of 32.41%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_1.NopolExampleTest#test9`
- `nopol_examples.nopol_example_1.NopolExampleTest#test8`
- `nopol_examples.nopol_example_1.NopolExampleTest#test7`
- `nopol_examples.nopol_example_1.NopolExampleTest#test6`
- `nopol_examples.nopol_example_1.NopolExampleTest#test5`
</details>